src/components/modals/export-map-modal/export-html-map.tsx

```diff
@@ -26,6 +26,9 @@ import {EXPORT_HTML_MAP_DOC, EXPORT_HTML_MAP_MODES_DOC} from 'constants/user-gui
 import styled from 'styled-components';
 import {injectIntl} from 'react-intl';
 import {FormattedMessage} from 'localization';
+import {IntlShape} from 'react-intl';
+
+import {setUserMapboxAccessToken, setExportHTMLMapMode} from 'actions';
```

Comment:
add a helper to `src/actions.ts` then used it to type all the action handlers in component
```js
export type ActionHandler<A extends (...args: any) => any> = (...args: Parameters<A>) => void;
```

use it here like this
```js
import {setUserMapboxAccessToken, setExportHTMLMapMode} from 'actions';

type ExportHtmlMapProps = {
  onChangeExportMapHTMLMode: ActionHandler<typeof setExportHTMLMapMode>;
  onEditUserMapboxAccessToken: ActionHandler<typeof setUserMapboxAccessToken>;
  ...
};
```